postmarketOS/pmbootstrap
# Fix invalid /etc/apk/repositories entry on device

It used to have an entry for /mnt/pmbootstrap-packages, which only makes sense while working on the chroot with pmbootstrap. After the installation on the device, there's no repo in that path.

This changes `pmbootstrap install`, as well as the Android recovery zip installation (therefore it would be cool if @ata2001 could take a look at it). I've also added four empty lines in the latter to make it more readable, is that okay?

Fixes #1186.

### How to test
* Do the normal installation
* Check if `/etc/apk/repositories` contains the `/mnt/pmbootstrap-packages` entry
* Do the Android recovery zip installation
* Check if `/etc/apk/repositories` contains the `/mnt/pmbootstrap-packages` entry
